CMakeLists.txt

```diff
@@ -0,0 +1,24 @@
+cmake_minimum_required(VERSION 3.19)
+
+set(CMAKE_CXX_STANDARD 20)
+set(CMAKE_CXX_STANDARD_REQUIRED on)
+
+if (APPLE)
+    foreach (HOMEBREW_PKG openssl libressl sqlite)
+        execute_process(COMMAND brew --prefix ${HOMEBREW_PKG} OUTPUT_VARIABLE HOMEBREW_PREFIX OUTPUT_STRIP_TRAILING_WHITESPACE)
+        list(APPEND CMAKE_PREFIX_PATH "${HOMEBREW_PREFIX}")
+    endforeach ()
+endif ()
+
+find_package(OpenSSL REQUIRED)
+find_package(SQLite3 REQUIRED)
+
+project(signalbackup-tools)
+
+file(GLOB_RECURSE SOURCES CONFIGURE_DEPENDS *.cc *.h)
+add_executable(signalbackup-tools ${SOURCES})
```

Comment:
No need to maintain a build script any longer. CMake automatically finds the source files and only recompiles the ones that were changed since last compilation.

Note that if a header file is modified, it only recompiles the source files that include that header file. It's very convenient compared to the current build system which doesn't recompile any `.cc` file if only the headers were modified (without CMake you need to manually delete the built objects to do that).